pouchdb-core/pouchdb-core.d.ts

```diff
@@ -268,6 +268,12 @@ declare namespace PouchDB {
         allDocs(options?: Core.AllDocsOptions):
             Promise<Core.AllDocsResponse<Content>>;
 
+        bulkDocs(docs: Core.Document<Content>[],
+                 options: Core.PutOptions | void,
```

Comment:
It looks like `bulkDocs` takes an `Array<Core.PutOptions>` (compare [`put` documentation](https://pouchdb.com/api.html#create_document) to [`bulkDocs` documentation](https://pouchdb.com/api.html#batch_create)).